x-pack/plugins/fleet/common/services/policy_template.ts

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Thanks for grouping all these utilities. Super helpful and understandable. Nice for tests as well.